Microsoft.Toolkit.Uwp.Notifications/Tiles/TileCommon.cs

```diff
@@ -15,21 +15,21 @@ public enum TileSize
         /// <summary>
         /// Small Square Tile
         /// </summary>
-        Small = 0,
+        Small = 1,
```

Comment:
Guess we'll want to document this as a breaking change too?

Reply:
Ah true that technically could be a breaking change